AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/README.md

```diff
@@ -53,55 +53,100 @@ You will need to download and install the following toolkits, tools, and compone
 Required AI Tools: <https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html>
 <br>If you have not already, select and install these Tools via [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html). AI and Analytics samples are validated on AI Tools Offline Installer. It is recommended to select Offline Installer option in AI Tools Selector.
 
+**2. (Offline Installer) Activate the AI Tools bundle base environment**
 
+If the default path is used during the installation of AI Tools:
+```
+source $HOME/intel/oneapi/intelpython/bin/activate
+```
+If a non-default path is used:
+```
+source <custom_path>/bin/activate
+```
+**3. (Offline Installer) Activate relevant Conda environment**
+
+```
+conda activate <offline-conda-env-name>
+``` 
+
+**4. Clone the GitHub repository**
+
+``` 
+git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/
+```
+
+**5. Install dependencies**
+``` 
+pip install -r requirements.txt

```

Comment:
I don't see requirements.txt file in the sample directory, we need to create such file. Additional libraries such as ipykernel should be included in this file
If there is no requirements.txt file and we still keep this command - it will lead to FileNotFoundError